Appraisals

```diff
@@ -8,13 +8,14 @@
 # > appraisal. If something is specified in both the Gemfile and an appraisal,
 # > the version from the appraisal takes precedence.
 # > https://github.com/thoughtbot/appraisal
-
+#
+#
 appraise "ar-5.2" do
-  gem "activerecord", "~> 5.2.4"
+  gem "rails", "~> 5.2.4"
```

Comment:
Should we maybe call the appraisals now `rails-5.2` etc, since it's not just `activerecord` (`ar`) anymore?

Reply:
Good idea